src/main/java/org/folio/rest/impl/BudgetAPI.java

```diff
@@ -42,7 +48,11 @@ public void getFinanceStorageBudgets(String query, String totalRecords, int offs
   @Override
   @Validate
   public void postFinanceStorageBudgets(Budget entity, Map<String, String> okapiHeaders, Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
-    PgUtil.post(BUDGET_TABLE, entity, okapiHeaders, vertxContext, PostFinanceStorageBudgetsResponse.class, asyncResultHandler);
+    RequestContext requestContext = new RequestContext(vertxContext, okapiHeaders);
+    budgetService.createBudget(entity, requestContext)
```

Comment:
Small suggestion we can save one line by refactoring (other similar places as well)
` budgetService.createBudget(entity, new RequestContext(vertxContext, okapiHeaders);)`